R/combineSNV_CNA.R

```diff
@@ -0,0 +1,146 @@
+# combineSNV_CNA.R
+
+#' Combine mutation data from CNA and SNV
+#'
+#' \code{combineSNV_CNA} loads the vector of preprocessed dataset from filtered CNA or SNV files,
+#'  creates hash keys corresponding to the input file type, combine the corresponding mutation count
+#'  to each gene and save to a gX format file as output file.
+#'
+#' Details.
+#' @section gX files:
+#'  gX rds files contain data of SNV and CNA combined for each gene as one object, sorted per
+#'  gene.
+#'
+#'
+#' @param fNameCNA A vector of path of fully qualified file names of SNV rds files.
```

Comment:
Also, consider changing the order of @param to match the order of the arguments in the function signature